tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td

```diff
@@ -17964,6 +17964,11 @@ On GPU, if an out of bound index is found, the index is ignored.
 
   TF_DerivedOperandTypeAttr T = TF_DerivedOperandTypeAttr<0>;
   TF_DerivedOperandTypeAttr Tindices = TF_DerivedOperandTypeAttr<1>;
+
+  let builders = [
```

Comment:
We need to check with some folks internally on whether local edits here make sense. May need to kick this back.